source/simulator/lateral_averaging.cc

```diff
@@ -312,22 +360,55 @@ namespace aspect
                                              std::vector<double>(n_slices,0.0));
     std::vector<double> volume(n_slices,0.0);
 
-    // this yields 10^dim quadrature points evenly distributed in the interior of the cell.
-    // We avoid points on the faces, as they would be counted more than once.
-    const QIterated<dim> quadrature_formula (QMidpoint<1>(),
-                                             10);
-    const unsigned int n_q_points = quadrature_formula.size();
+    // We would like to use a quadrature formula that is appropriately accurate laterally,
+    // but has a high resolution in depth (to populate all depth slices for the averages).
```

Comment:
Consider adding in a brief explanation of why there could be more than one depth slice in a cell (AMR).